woocommerce/woocommerce-android
# Issue/10555 retrieve campaign data logic

Closes: #10555


### Description

Retrieves campaign preview default data from repository and removes hardcoded values. Additionally, it ensures that campaign details section is displayed while the ad details are being loaded.

### Testing instructions

Click on "Create campaign" button and check that details section is visible with campaign default values while AI suggestions are being loaded (for now, loading is just a simulation using a `delay`)

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/2663464/1d9bd46b-686a-4df9-a004-9c814a5049b0
